### PR TITLE
Update docs & test coverage

### PR DESCRIPTION
**Pull request by dimitraz**
**Describe what this PR does and why we need it**:
Improve test coverage and command descriptions for the CLI.

Changes proposed in this pull request
 - Create tests for `list service instances` command
 - Clean up existing tests
 - Update command descriptions
 - Add/modify license headers

@pyates86

### CODE REVIEW

**Comment by dimitraz** [comment]
@JameelB

File: cmd/mobile/main.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright © 2017 NAME HERE <EMAIL ADDRESS>
+// Copyright Red Hat, Inc., and individual contributors.
```
Comment (JameelB) [inline cmd/mobile/main.go]: Do we need the license within the start of each file? I think we already have an Apache license on the overall repo. @maleck13

Comment (maleck13) [reply cmd/mobile/main.go]: not entirely sure either. Does not real harm I guess

Comment (dimitraz) [reply cmd/mobile/main.go]: I guess I'll remove them in that case. It was more so for just keeping consistency

Comment (maleck13) [reply cmd/mobile/main.go]: No leave them for now

Comment (dimitraz) [reply cmd/mobile/main.go]: Ok

**Comment by maleck13** [comment]
can you squash these commits down to 1 or 2 logical commits

**Comment by dimitraz** [comment]
@maleck13 done

File: pkg/cmd/services_test.go [hunk 2]
```diff
@@ -442,11 +455,128 @@ func TestServicesCmd_CreateServiceInstanceCmd(t *testing.T) {
...
+			K8Client: func() kubernetes.Interface {
+				return &kFake.Clientset{}
+			},
+			ValidateOut: func(t *testing.T, data []byte) {
+				var list = &v1beta1.ServiceInstanceList{}
+
+				if err := json.Unmarshal(data, list); err != nil {
+					t.Fatal("failed to unmarshal data", err)
+				}
+				if nil == list {
```
Comment (maleck13) [inline pkg/cmd/services_test.go]: I don't think list would ever be nil here as it has been allocated on line 498 so can probably remove this

Comment (dimitraz) [reply pkg/cmd/services_test.go]: Good point, thanks

**Pull request merged**